oslc_prolog/lib/oslc_core.pl

```diff
@@ -19,24 +20,38 @@
 
 get_current_time(Context) :-
   get_time(T),
-  make_temp_graph(Context.graph_out),
   create_resource(oslc:time, [oslc:'Time'],
-                 [dcterms:created='^^'(T, xsd:dateTime)], rdf(Context.graph_out)).
+                 [dcterms:created='^^'(T, xsd:dateTime)], tmp(Context.graph_out)).
 
 get_time_class(Context) :-
-  make_temp_graph(Context.graph_out),
-  create_resource(oslc:'Time', [rdfs:'Class'], [], rdf(Context.graph_out)).
+  create_resource(oslc:'Time', [rdfs:'Class'], [], tmp(Context.graph_out)).
 
 handle_ontology(Context) :-
-  once((
+  findall(Graph, (
```

Comment:
FEATURE: When GETting an ontology by `http://<host>/<prefix>`, and multiple name graphs have a name that starts with URI of the `<prefix>`, the response will contain resources from all of them (see `pddle` ontology for example).